bridge/bridge.go

```diff
@@ -181,15 +224,14 @@ func (b *Bridge) newService(port ServicePort, isgroup bool) *Service {
 	}
 
 	// not sure about this logic. kind of want to remove it.
-	hostname, err := os.Hostname()
-	if err != nil {
+	hostname := Hostname
+	if hostname == "" {
 		hostname = port.HostIP
-	} else {
-		if port.HostIP == "0.0.0.0" {
-			ip, err := net.ResolveIPAddr("ip", hostname)
```

Comment:
Yeah, I agree. We explicitly pass in a `-h` for our registrator, and the existing behavior used to be to leave it blank.

But again, that's outside the scope of this PR. This is the current behavior. :)